src/components/ProductCard.tsx

```diff
@@ -1,89 +1,86 @@
-import React, { useEffect, useState } from "react";
-import { Transition } from "@headlessui/react";
+import React, { useState } from "react";
 import Image from "next/image";
 import ProudctNameAndEdition from "./ProductNameAndEdition";
 import ProductColourButton from "./ProductColourButton";
+import ProductCardShowMoreButton from "./productCardShowMoreButton";
 
 export type Content = {
   colours: string[];
   image: string[];
+  imageAlt: string;
+  index: number;
```

Comment:
I don't see this being used, am I missing something here? is there a reason for index to be part of the Content object?

Reply:
index is removed due to not being used. ImageAlt is being used ProductCardShowMoreButton when passing content props.